assets/src/edit-story/components/canvas/pagemenu/index.js

```diff
@@ -249,40 +239,29 @@ function PageMenu() {
               style={{ marginLeft: 'auto' }}
               title={__('Stop', 'web-stories')}
             >
-              <Icon
+              <PageMenuButton
                 onClick={toggleAnimationState}
                 disabled={!hasAnimations}
                 aria-label={__('Stop Page Animations', 'web-stories')}
               >
-                <StopCircular />
-              </Icon>
+                <Icons.StopAnimation />
+              </PageMenuButton>
             </WithTooltip>
           ) : (
             <WithTooltip
               style={{ marginLeft: 'auto' }}
               title={__('Play', 'web-stories')}
             >
-              <Icon
+              <PageMenuButton
                 onClick={toggleAnimationState}
                 disabled={!hasAnimations}
                 aria-label={__('Play Page Animations', 'web-stories')}
               >
-                <PlayCircular />
-              </Icon>
+                <Icons.PlayAnimation />
+              </PageMenuButton>
             </WithTooltip>
           )}
         </Options>
-        {showTextMagicAndHelperMode && (
-          <Options>
-            <Icon disabled>
-              <LayoutHelper />
-            </Icon>
-            <Space isDouble />
-            <Icon disabled>
-              <Text />
-            </Icon>
-          </Options>
-        )}
```

Comment:
I decided to fully remove this (but not the underlying flag) as it was just some old icons lying around with no accompanying functionality. If we ever need this functionality, design has to be redone anyway.

@swissspidy, should I remove the flag as well?

Reply:
Yeah let's also remove the flag.

Reply:
Removed in 0a21ace.